tutorials/editor/project_settings.rst

```diff
@@ -1,16 +1,138 @@
+**Project Settings** window is available to edit these settings. To open the
+Project Settings, select **Project > Project Settings** from the main menu.
+
+.. figure:: img/project_settings_basic.webp
+    :align: center
+
+    The Project Settings window
+
+The **Project Settings** window is mainly used to change settings in the
+**General** tab. Additionally, there are tabs for the
+:ref:`Input Map <doc_input_examples_input_map>`,
+:ref:`Localization <doc_internationalizing_games>`,
+:ref:`Globals <doc_singletons_autoload>`,
+:ref:`Plugins <doc_installing_plugins_enabling_a_plugin>`, and
+**Import Defaults**. Usage of these other tabs is documented elsewhere.
+
+Changing project settings
+-------------------------
+
+The **General** tab of the project settings window works much like the inspector.
+It displays a list of project settings which you can change, just like inspector
+properties. There is a list of categories on the left, which you can use to select
+related groups of settings. You can also search for a specific setting with the
+**Filter Settings** field.
+
+Each setting has a default value. Settings can be reset to their default values
+by clicking the circular arrow **Reset** button next to each property.
+
+Changing project settings from code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use :ref:`set_setting() <class_ProjectSettings_method_set_setting>` to
+change a setting's value from code:
+
+.. tabs::
+    .. code-tab:: gdscript GDScript
+        
+        ProjectSettings.set_setting("application/run/max_fps", 60)
+        ProjectSettings.set_setting("display/window/size/mode", DisplayServer.WINDOW_MODE_WINDOWED)
+
+    .. code-tab:: csharp
+
+        ProjectSettings.SetSetting("application/run/max_fps", 60);
+        ProjectSettings.SetSetting("display/window/size/mode", (int)DisplayServer.WindowMode.Windowed);
+
+However, many project settings are only read once when the game starts. After
+that, changing the setting with ``set_setting()`` will have no effect. Instead, 
+most settings have a corresponding property or method on a runtime class like
+:ref:`Engine <class_Engine>` or :ref:`DisplayServer <class_DisplayServer>`:
+
+.. tabs::
+    .. code-tab:: gdscript GDScript
+        
+        Engine.max_fps = 60
+        DisplayServer.window_set_mode(DisplayServer.WINDOW_MODE_WINDOWED)
+
+    .. code-tab:: csharp
+
+        Engine.MaxFps = 60;
+        DisplayServer.WindowSetMode(DisplayServer.WindowMode.Windowed);
+
+In general, project settings are duplicated at runtime in the
+:ref:`Engine <class_Engine>`, :ref:`PhysicsServer2D <class_PhysicsServer2D>`,
+:ref:`PhysicsServer3D <class_PhysicsServer3D>`,
+:ref:`RenderingServer <class_RenderingServer>`, 
+:ref:`Viewport <class_Viewport>`, or :ref:`Window <class_Window>` classes. In the
+:ref:`ProjectSettings <class_ProjectSettings>` class reference, settings
+links to their equivalent runtime property or method.
```

Comment:
This was originally "each project setting links to its runtime equivalent" but that's not true until https://github.com/godotengine/godot-docs/issues/9944 is done...